pkg/fs/apkfs.go

```diff
@@ -235,8 +235,17 @@ func (a *APKFS) Stat(path string) (fs.FileInfo, error) {
 	}
 	return &apkFSFileInfo{file: file, name: file.name[strings.LastIndex(file.name, "/"):]}, nil
 }
-
+func correctApkFSPath(path string) string {
+	if path == "." {
+		path = "/"
+	}
+	if path[:2] == "./" {
```

Comment:
You need to verify the length of path before taking a slice of this length or you will cause a panic.